packages/types/src/extrinsic/signedExtensions/index.ts

```diff
@@ -5,16 +5,19 @@ import type { ExtDef, ExtInfo, ExtTypes } from './types';
 
 import { polkadot } from './polkadot';
 import { shell } from './shell';
+import { statemint } from './statemint';
 import { substrate } from './substrate';
 
 // A mapping of the known signed extensions to the extra fields that they contain. Unlike in the actual extensions,
 // we define the extra fields not as a Tuple, but rather as a struct so they can be named. These will be expanded
 // into the various fields when added to the payload (we only support V4 onwards with these, V3 and earlier are
 // deemed fixed and non-changeable)
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Weird that this is needed, I cannot spot why.

Reply:
maybe my vscode picked up a weird default config for eslint?